### PR TITLE
Support AGB

### PR DESCRIPTION
**Pull request by RamiroRama**
Hi guys! I make this pull request so that you please add the logo of the token that we have created, thank you.

### CODE REVIEW

File: src/rest/usePairs.ts [hunk 1]
```diff
@@ -304,6 +304,17 @@ export default () => {
               ) {
                 return
               }
+              
+              if (
+                networkName === "mainnet" &&
+                tokenInfo2?.symbol === "AGB" &&
```
Comment (JoowonYun) [inline src/rest/usePairs.ts:310]: It seems that you need content about tokenInfo1 as well. Also, if the repo(https://github.com/terra-money/assets) is whitelisted, the code is not needed.

Comment (RamiroRama) [reply src/rest/usePairs.ts:310]: I don't really know what this code modification does, I saw it in some commits that I was reading while studying how to make this contribution on github, I thought that the code could contribute the logo to the lp. If it doesn't work for this case, let's remove that modification.

Comment (JoowonYun) [reply src/rest/usePairs.ts:310]: This part is a condition for white listing when there is the same symbol. In pair, token1 and token2 exist, and currently only token2 is processed. Add the same IF statement for tokenInfo1.

If you don't want to do this, you're following the whitelisting of the `asset` repo currently operate by the Terra Foundation.

Comment (RamiroRama) [reply src/rest/usePairs.ts:310]: Would it be okay if i delete this pair.ts file? so It can advance the commit?

Comment (JoowonYun) [reply src/rest/usePairs.ts:310]: If you are not concerned about tokens with the same AGB symbol, you can proceed as is.

Comment (RamiroRama) [reply src/rest/usePairs.ts:310]: There is another token with the same symbol at this moment, it was another test that I did, of that I will not build LP nor will I do it. But you tell me what would be the best thing to do and I do it.

Comment (RamiroRama) [reply src/rest/usePairs.ts:321]: I just added the conditional with the infotoken1

**Review by JoowonYun — APPROVED** [review]

**Comment by RamiroRama** [comment]
Thank you very much for your help and patience!

**Pull request merged**